services/tenant-ui/frontend/src/components/common/LocaleSwitcher.vue

```diff
@@ -8,7 +8,7 @@ import { useI18n } from 'vue-i18n';
 
 const { locale } = useI18n({ useScope: 'global' });
 
-const localeList = ['en', 'fr'];
+const localeList = ['en', 'fr', 'jp'];
```

Comment:
Just curious... Is this Japanese? If so, do we need to support this?

Reply:
Yeah, it's just a sample language though, not going to do any of the translations at this point, wanted one to demo that has non English characters though.